### PR TITLE
Allow adding multiple issues to a project 

### LINKED ISSUE #17216: [Feature] Allow "mass-assign" issue to a project
I would like to suggest that gitea allows "mass-assign" issues to a project, similar to labels. 

You can just select multiple issues and choose a label for all of them. Similarly, I would like to be able to select multiple issues and assign them to one project at once.

Edit: Clarified what I meant.

### PR DESCRIPTION
**Pull request by qwerty287**
Adds an option like for adding multiple issues to a milestone.
Closes #17216

### CODE REVIEW

**Review by delvh — CHANGES_REQUESTED** [review]
I tested it out.
Nothing seems to have changed?
Or where is the template repo/issue/list.tmpl shown?
I would have guessed that to be the sidebar to modify issue attributes, but apparently that's not the case…

**Comment by qwerty287** [comment]
@delvh it's used in the repo list {owner}/{repo}/issues 
If you use the checkboxes next to the issues, you'll find the new option "projects" in the top bar (along with close, milestone, labels...). If you select a label there, all selected issues will be added to this project.

File: templates/repo/issue/list.tmpl [hunk 1]
```diff
@@ -166,6 +166,24 @@
...
 					</div>
 
+					<!-- Projects -->
+					<div class="ui {{if not .Projects}}disabled{{end}} dropdown jump item">
+						<span class="text">
+							{{.i18n.Tr "repo.project_board"}}
+							{{svg "octicon-triangle-down" 14 "dropdown icon"}}
+						</span>
+						<div class="menu">
+							<div class="item issue-action" data-element-id="0" data-url="{{$.Link}}/milestone">
```
Comment (delvh) [inline templates/repo/issue/list.tmpl]: Copy-paste error?
```suggestion
							<div class="item issue-action" data-element-id="0" data-url="{{$.Link}}/projects">
```

File: routers/web/repo/issue.go [hunk 2]
```diff
@@ -327,6 +327,20 @@ func issues(ctx *context.Context, milestoneID, projectID int64, isPullOption uti
 		}
 		return 0
 	}
+
+	if ctx.Repo.CanWriteIssuesOrPulls(ctx.Params(":type") == "pulls") {
```
Comment (delvh) [inline routers/web/repo/issue.go:331]: That query param was already present and wasn't added in this PR, right?

Comment (qwerty287) [reply routers/web/repo/issue.go:331]: Yes, it was already available (and is used in `func Issues` few lines below)

**Review by delvh — APPROVED** [review]

**Review by zeripath — APPROVED** [review]

**Comment by 6543** [comment]
:rocket:

**Pull request merged**